### ISSUE TITLE
Templates Pagination not functioning 

### ISSUE THREAD

**Issue by bakobako** [klaviyo/klaviyo-api-python#6]
With the old SDK the Get all templates call was with page count pagination : 
https://github.com/klaviyo/klaviyo-python-sdk#templates
https://developers.klaviyo.com/en/v1-2/reference/get-templates

With the new SDK (and api) Get all templates is paginated with the cursor : https://developers.klaviyo.com/en/reference/get_templates
https://github.com/klaviyo/klaviyo-api-python#get-templates

I have 29 templates in my Klaviyo account. fetching with the old client/api i get 29. 

Fetching with the new client/api I get only the 20 from the first page and the next cursor is none:

`'links' : {'self': 'https://a.klaviyo.com/api/templates/', 'next': None, 'prev': None}`

is this an issue with the API or SDK or is there something im doing wrong?

I just run 

```
client = KlaviyoAPI(api_token, max_delay=60, max_retries=5)
first_page = client.Templates.get_templates()
```

and the first page has the above links with no next page

**Comment by jon-batscha** [comment]
Hey! Appreciate you reaching out. 

I just tried to recreate your issue and was not able to. Can you try running the following script:

```python
from klaviyo_api import KlaviyoAPI

klaviyo = KlaviyoAPI("YOUR_API_KEY")
templates = klaviyo.Templates.get_templates()
print(len(templates['data']))
pages=1
template_count=len(templates['data'])
while True:
    print(len(templates['data']))
    template_count += len(templates['data'])
    page_cursor = templates['links']['next']
    if page_cursor == None:
        break
    templates = klaviyo.Templates.get_templates(page_cursor=page_cursor)
    pages+=1
print('PAGES:',pages)
print('TEMPLATES:',template_count)
```

**Comment by bakobako** [comment]
@jon-batscha any update on this please?

**Comment by jon-batscha** [comment]
Hey!

Following up on this, just tried again with your script and was unable to recreate your issue. Specifically, in addition to testing with my own script, i ran yours, and got the following output:

```
20
20
15
PAGES: 2
TEMPLATES: 35
Now validating old sdk
total based on api: 35
total based on api: 35
TEMPLATES: 35
```

**Comment by jon-batscha** [comment]
Can you try re-running your script with the latest version to confirm that the issue is fixed?

**Comment by bakobako** [comment]
Hi @jon-batscha I used the newest v1.2.1 and I did some further testing. I added 2 more templates;
1 where I duplicated an existing template without changing the name and 
1 where I duplicated the template and changed the name.

My results were:

20
20
1
PAGES: 2
TEMPLATES: 21
Now validating old sdk
total based on api: 31
total based on api: 31
TEMPLATES: 31

This means that the old API and the UI of klaviyo show all 31 and the new api seems to only return values with unique names. 

Which is strange, if it deduplicates on the names of templates instead of IDs.
